lib/client/debug/debug.go

```diff
@@ -55,10 +56,14 @@ func NewClient(socketPath string) *Client {
 		clt: &http.Client{
 			Timeout: apidefaults.DefaultIOTimeout,
 			Transport: &http.Transport{
```

Comment:
Do we care about setting `IdleConnTimeout`? We never call `CloseIdleConnection` on the http client or transport, but this should always get used in a short-lived process so maybe there's no reason to be concerned?

Reply:
Thinking about it slightly more - we should probably just set `DisableKeepAlives` and let the transport open a new connection every time, or we risk potentially staying connected to a "stale" `teleport` process while it's shutting down and never actually hitting the new one.

Reply:
Added